uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/Button.java

```diff
@@ -50,6 +59,8 @@ public void setType(final ButtonType type) {
     }
 
     public void setButtonStyleType(final ButtonStyleType type) {
+        removeClass("btn");
```

Comment:
Just question, why we remove this "btn" class when changing button style, but we declare this class by default, for example in `ConfirmPopup.html`:
```html
<button type="button" class="btn btn-default" data-field="confirm-cancel" data-dismiss="modal"></button>
<button type="button" class="btn btn-danger" data-field="confirm-ok"></button>
```

Reply:
The "btn" class is also removed for safety and to be sure that all the elements that configure the button style are properly set to the new style.